Open Judge System/Workers/OJS.Workers.ExecutionStrategies/BlockchainStrategies/GanacheCliScope.cs

```diff
@@ -0,0 +1,166 @@
+﻿namespace OJS.Workers.ExecutionStrategies.BlockchainStrategies
+{
+    using System;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+
+    using OJS.Common;
+
+    internal class GanacheCliScope : IDisposable
```

Comment:
Two questions here. Why `internal` and why `Scope`?

Reply:
`internal` because it is meant to be used only in the execution strategies assembly, more specifically in the solidity strategies, where tests have to be executed in the scope of the ethereum virtual machine, which is created by ganache-cli. `Scope` might be removed, since GanacheCli seems descriptive enough used in a `using{}` block